runtime/common/src/claims.rs

```diff
@@ -1268,7 +1287,13 @@ mod benchmarking {
 
 		// Benchmark the time it takes to execute `validate_unsigned` for `claim`
 		validate_unsigned_claim {
```

Comment:
Aren't signed extension not executed during benchmarks?

Reply:
if benchmark use `_($args)` or `$dispatch($args)` then it only execute:
```rust
				<
					Call<T $(, $instance)? > as $crate::frame_support::traits::UnfilteredDispatchable
				>::dispatch_bypass_filter(
					Call::<T $(, $instance)? >::$dispatch($($arg),*), $origin.into()
				)?;
```

Reply:
def not. the only logic executed during the benchmark is what is define in the dispatchable function or whatever other logic the benchmark wraps